packages/api/cms-api/src/dam/files/entities/license.embeddable.ts

```diff
@@ -0,0 +1,46 @@
+import { Embeddable, Enum, Property } from "@mikro-orm/core";
+import { Field, ObjectType, registerEnumType } from "@nestjs/graphql";
+
+export enum LicenseType {
+    ROYALTY_FREE = "royalty_free",
+    RIGHTS_MANAGED = "rights_managed",
+    SUBSCRIPTION = "subscription",
+    MICRO = "micro",
```

Comment:
I believe Enums in GraphQL require the same key as value. At least it did in a previous Nest version. We should check if this is still the case.

Reply:
I didn't encounter any problems when testing, but I still changed it to uppercase